### DISCUSSION TITLE
How to Play DASH format video in Nextjs

### DISCUSSION THREAD

**Discussion by mokshith-moksh** [muxinc/media-chrome#897] [Q&A] [answered]
I am trying to implement , in nextjs I found out this problem ,

**Comment by luwes** [comment, marked as answer]
We have a custom video element for dash.js.
https://github.com/muxinc/media-elements/tree/main/packages/dash-video-element

example here in Next.js 
https://media-elements-nextjs.vercel.app/dash-video
https://github.com/muxinc/media-elements/blob/main/examples/nextjs/app/dash-video/page.tsx
